cmd/createcluster_internal_test.go

```diff
@@ -146,6 +187,21 @@ func testCreateCluster(t *testing.T, conf clusterConfig) {
 		require.NoError(t, json.Unmarshal(b, &lock))
 		require.NoError(t, lock.VerifyHashes())
 		require.NoError(t, lock.VerifySignatures())
+
+		if conf.DefFile != "" {
+			var def cluster.Definition
+			def, err = loadDefinition(context.Background(), conf.DefFile)
+			require.NoError(t, err)
+
+			// Config hash and creator should remain the same
+			require.Equal(t, def.ConfigHash, lock.ConfigHash)
```

Comment:
rather create variable for these above, then no need to load the definition again.

Reply:
yeah makes sense as we already create a definition which can be passed to `testCluster` and used here